influxdb3/src/commands/serve.rs

```diff
@@ -372,6 +377,30 @@ pub async fn command(config: Config) -> Result<()> {
     Ok(())
 }
 
+async fn setup_telemetry_store(
+    object_store_config: &ObjectStoreConfig,
+    instance_id: Arc<str>,
+    num_cpus: usize,
+) -> Arc<TelemetryStore> {
+    let os = std::env::consts::OS;
+    let influxdb_pkg_version = env!("CARGO_PKG_VERSION");
```

Comment:
will these be present in the built binary?

Reply:
Yes - these are available for all rust packages. I'll add another field that explicitly captures the type of the product like oss or not.